CMakeLists.txt

```diff
@@ -19,15 +13,19 @@ endif()
 enable_testing()
 
 # Leatherman setup
-SET(LEATHERMAN_COMPONENTS locale catch nowide logging)
+SET(LEATHERMAN_COMPONENTS locale catch nowide logging util file_util execution)
+
+if (WIN32)
+    list(APPEND LEATHERMAN_COMPONENTS windows)
+endif()
+
 find_package(Leatherman REQUIRED COMPONENTS ${LEATHERMAN_COMPONENTS})
 
-# Now that we have leatherman, we can pull in its options file, which
-# we need for finding all our other libraries.
+# Pull in leatherman options
 include(options)
-## Pull in common cflags setting from leatherman. Don't override CMAKE_CXX_FLAGS at the project root to avoid impacting 3rd party code.
+# Pull in common cflags setting from leatherman.
 include(cflags)
-set(${PROJECT_NAME_UPPER}_CXX_FLAGS "${LEATHERMAN_CXX_FLAGS}")
+set(WHEREAMI_CXX_FLAGS "${LEATHERMAN_CXX_FLAGS}")
```

Comment:
If you find it easier to read, that can be a reason. I was just asking to understand. (When I said it should be identical, I meant there's no functional change here).